app/_data/docs_nav_konnect.yml

```
@@ -326,6 +312,18 @@
           absolute_url: true
           generate: false
           target_blank: true
+    - text: Deprecated APIs
+      items:
+        - text: (Deprecated) - Runtime Configuration API
```

```suggestion
        - text: (Deprecated) Runtime Configuration API
```
I feel like it doesn't need that extra character